zeppelin-web/README.md

```diff
@@ -19,19 +19,27 @@ It is recommended to install node 6.0.0+ since Zeppelin uses 6.9.1+ (see [creati
 
 All build commands are described in [package.json](./package.json)
 
-```sh
 # install required depepdencies and bower packages (only once)
-$ npm install -g yarn
-$ yarn install
+```
+$ npm install -g yarn bower npm-run-all rimraf grunt webpack karma karma-phantomjs-launcher
```

Comment:
Thanks for doc reference about call script with relative path. In my opinion this is will be a problem because path delimeters are different for Linux and Windows and we should not make zeppelin is platform dependent